utils/atheepmgr/Makefile

```
@@ -0,0 +1,71 @@
+#
+# Copyright (c) 2021 Sergey Ryazanov <ryazanov.s.a@gmail.com>
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=atheepmgr
+PKG_VERSION:=2.1.1
+PKG_RELEASE:=1
```

It is up to you if you want to use autorelease or not

```suggestion
PKG_RELEASE:=$(AUTORELEASE)
```

---

Since I maintain both the utility themself and the package and since package a quite simple, release will not change too often if any.